webui/react/src/components/ThemeToggle.test.tsx

```diff
@@ -0,0 +1,52 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { Router } from 'react-router-dom';
+
+import StoreProvider from 'contexts/Store';
+import history from 'shared/routes/history';
+
+import ThemeToggle from './ThemeToggle';
+
+const ThemeToggleContainer: React.FC = () => {
+
+  return (
+    <StoreProvider>
+      <Router history={history}>
+        <ThemeToggle />
+      </Router>
+    </StoreProvider>
+  );
+};
+
+const SYSTEM_MODE = 'System Mode';
+const LIGHT_MODE = 'Light Mode';
+const DARK_MODE = 'Dark Mode';
+
+const setup = () => render(<ThemeToggleContainer />);
+
+describe('ThemeToggle', () => {
+  it('System Mode is the default setting', async () => {
```

Comment:
**nit:** for our `it` tests, we want to write like like a sentence. So for this example, something like this would read better:
```
it('should have System Mode as the default setting', async () => {
```